src/graphql/resolvers/queries.js

```diff
@@ -89,9 +90,16 @@ module.exports = {
   // attr info give informations about graphql request
   resolve: async (parent, args, context) => {
     let index = context?.get('index');
+    const apikey = context?.get('x-api-key');
+
+    const configApi = await redisClient.get(apikey);
+    const parsedConfigApi = JSON.parse(configApi);
+    const { attributes } = parsedConfigApi;
```

Comment:
The key metadata is already fetched in a middleware. You could attach it to the `request` in the middleware, so that you can access it in the `context` here.